web/src/user_groups_settings_ui.js

```diff
@@ -54,13 +54,13 @@ export const show_user_group_settings_pane = {
         );
     },
     settings(group) {
-        $(".settings, #user-group-creation").hide();
+        $(".nothing-selected, #user-group-creation").hide();
         $("#groups_overlay .settings").show();
         set_active_group_id(group.id);
         $("#groups_overlay .user-group-info-title").text(group.name);
     },
     create_user_group() {
-        $(".nothing-selected, .settings, #user-group-creation").hide();
+        $(".nothing-selected, .settings").hide();
```

Comment:
Both `.settings` and the `.nothing-selected` selector seem badly scoped -- we want to just interact with the user groups variant of these, not other elements...,

Reply:
Fixed this to only query these classes within `#groups_overlay`.